src/styles/_variables.scss

```diff
@@ -14,7 +9,7 @@ $data-viewer-border-color: #e0e0e0 !default;
 $grey: #eeeeee;
 $grey-dark: #585858;
 
-//GLOBAL STYLE
+//GLOBAL STYLE (by defautlt we use the google font "Montserrat"
```

Comment:
```suggestion
// Global styles (by default, we use the google font "Montserrat")
```

Reply:
I think the styles below should be part of the DataViewer and QueryBuilder components, not in this global file.
Is there something preventing that?